crates/rattler_conda_types/src/prefix_record.rs

```diff
@@ -234,8 +235,41 @@ impl PrefixRecord {
         path: impl AsRef<Path>,
         pretty: bool,
     ) -> Result<(), std::io::Error> {
-        let file = File::create(path.as_ref())?;
-        self.write_to(BufWriter::with_capacity(50 * 1024, file), pretty)
+        const BUFFER_SIZE: usize = 64 * 1024;
```

Comment:
What is this based on?

Reply:
Used to be 50 * 1024, now it's 64 * 1024 - doesn't really matter that much but it saves us some calls to `write` by having a slightly larger buffer. This buffer size is 64Kb.